packages/nimble-components/src/table/styles.ts

```diff
@@ -85,9 +87,17 @@ export const styles = css`
         pointer-events: none;
```

Comment:
I can't recall, did we talk with designers about what the mouse cursor should be when hovering over a selectable row? The current behavior of an arrow cursor for the row matches what SLE does today and what [sharepoint's file browser](https://nio365.sharepoint.com/sites/SystemLinkDesignSystem/Shared%20Documents/Forms/AllItems.aspx) does, so it may be correct. It just seemed a bit surprising to me not to have cursor feedback that it is operable.

Reply:
Brought this up with the designers, and they want to keep the default mouse cursor. Therefore, nothing to be done on this.